### PR TITLE
feat: delete multiple entities; save credentials default true;

### PR DESCRIPTION
**Pull request by bobvandenhoogen**
strm context billing-id shows no login information if user is not logged in

### CODE REVIEW

File: pkg/entity/batch_exporter/cmd.go [hunk 1]
```diff
@@ -18,17 +18,20 @@ const (
 
 func DeleteCmd() *cobra.Command {
 	batchExporter := &cobra.Command{
-		Use:   "batch-exporter [name]",
-		Short: "Delete a Batch exporter by name",
+		Use:   "batch-exporter [name, ...]",
```
Comment (trietsch) [inline pkg/entity/batch_exporter/cmd.go:21]: Comma separated? Unix style is spaces, so I think we should adhere to that.

Comment (trietsch) [reply pkg/entity/batch_exporter/cmd.go:21]: Maybe also include that in the help text, that it should be space separated.

File: pkg/entity/kafka_user/cmd.go [hunk 2]
```diff
@@ -12,15 +12,17 @@ const (
 
 func DeleteCmd() *cobra.Command {
 	return &cobra.Command{
-		Use:   "kafka-user [name]",
-		Short: "Delete a Kafka user",
+		Use:   "kafka-user [name, ...]",
+		Short: "Delete on ore more Kafka users",
```
Comment (trietsch) [inline pkg/entity/kafka_user/cmd.go:16]: typo `ore` -> `one`

File: pkg/entity/kafka_user/cmd.go [hunk 3]
```diff
@@ -12,15 +12,17 @@ const (
 
 func DeleteCmd() *cobra.Command {
 	return &cobra.Command{
-		Use:   "kafka-user [name]",
-		Short: "Delete a Kafka user",
+		Use:   "kafka-user [name, ...]",
```
Comment (trietsch) [inline pkg/entity/kafka_user/cmd.go:15]: Same, space separated?

File: pkg/entity/stream/cmd.go [hunk 4]
```diff
@@ -48,10 +48,10 @@ func completion(cmd *cobra.Command, args []string, complete string) ([]string, c
 }
 
 func DeleteCmd() *cobra.Command {
-	return &cobra.Command{
-		Use:   "stream [name]",
-		Short: "Delete a stream",
-		Long: `Delete a stream.
+	stream := &cobra.Command{
+		Use:   "stream [name, ...]",
```
Comment (trietsch) [inline pkg/entity/stream/cmd.go:52]: Same, space separated.

File: pkg/context/cmd.go [hunk 5]
```diff
@@ -49,6 +51,9 @@ func BillingIdInfo() *cobra.Command {
 			printer = configurePrinter(cmd)
 		},
 		Run: func(cmd *cobra.Command, args []string) {
+			if len(strings.TrimSpace(auth.Auth.BillingId())) == 0 {
```
[the remark this answers is not in the document]
Comment (bobvandenhoogen) [reply pkg/context/cmd.go:54]: authenticator is undefined. Will replace with 
```
- = auth.Auth.BillingId()
```
which has this same logic

**Review by trietsch — APPROVED** [review]

**Pull request merged**